docs/configuration/SUMMA_model_decisions.md

```diff
@@ -3,3 +3,396 @@
+##  2. simulFinsh
+Simulation end time
+
+End of the simulation specified as `'YYYY-MM-DD hh:mm'`. Note that the string needs to be enclosed in single quotes. This indicates the end of the last time step.
+
+<a id="soilCatTbl"></a>
+##  3. soilCatTbl
+Soil-category dataset
+
+| Option | Description |
+|---|---|
+| STAS | **TODO: Describe STAS <br> [Reference](http://doi.org/)** |
+| STAS-RUC | **TODO: Describe STAS-RUC <br> [Reference](http://doi.org/)** |
+| ROSETTA | **TODO: Describe ROSETTA <br> [Reference](http://doi.org/)** |
+
+
+<a id="vegeParTbl"></a>
+##  4. vegeParTbl
+Vegetation category dataset
+
+| Option | Description |
+|---|---|
+| USGS | **TODO: Describe USGS <br> [Reference](http://doi.org/)** |
+| MODIFIED_IGBP_MODIS_NOAH | **TODO: Describe MODIFIED_IGBP_MODIS_NOAH <br> [Reference](http://doi.org/)** |
+| plumberCABLE | **TODO: Describe plumberCABLE <br> [Reference](http://doi.org/)** |
+| plumberCHTESSEL | **TODO: Describe plumberCHTESSEL <br> [Reference](http://doi.org/)** |
+| plumberSUMMA | **TODO: Describe plumberSUMMA <br> [Reference](http://doi.org/)** |
+
+<a id="soilStress"></a>
+##  5. soilStress
+Function for the soil moisture control on stomatal resistance
+
+| Option | Description |
+|---|---|
+| NoahType | Soil stress is a thresholded linear function of volumetric liquid water content <br> **TODO: Describe NoahType <br> [Reference](http://doi.org/)** |
+| CLM_Type | Soil stress is a thresholded linear function of matric head <br> **TODO: Describe CLM_Type <br> [Reference](http://doi.org/)** |
+| SiB_Type | Soil stress is an exponential of the log of matric head <br> **TODO: Describe Sib_Type <br> [Reference](http://doi.org/)** |
+
+<a id="stomResist"></a>
+##  6. stomResist
+Function for stomatal resistance
+
+| Option | Description |
+|---|---|
+| BallBerry | **TODO: Describe BallBerry <br> [Reference](http://doi.org/)** |
+| Jarvis | **TODO: Describe Jarvis <br> [Reference](http://doi.org/)** |
+| simpleResistance | **TODO: Describe simpleResistance <br> [Reference](http://doi.org/)** |
+| BallBerryFlex | **TODO: Describe BallBerryFlex <br> [Reference](http://doi.org/)** |
+| BallBerryTest | **TODO: Describe BallBerryTest <br> [Reference](http://doi.org/)** |
+
+<a id="bbTempFunc"></a>
+##  7. bbTempFunc
+Ball-Berry: leaf temperature controls on photosynthesis + stomatal resistance
+
+| Option | Description |
+|---|---|
+| q10Func | **TODO: Describe q10Func <br> [Reference](http://doi.org/)** |
+| Arrhenius | **TODO: Describe Arrhenius <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbHumdFunc"></a>
+##  8. bbHumdFunc
+Ball-Berry: humidity controls on stomatal resistance
+
+| Option | Description |
+|---|---|
+| humidLeafSurface | **TODO: Describe humidLeafSurface <br> [Reference](http://doi.org/)** |
+| scaledHyperbolic | **TODO: Describe scaledHyperbolic <br> [Reference](http://doi.org/)** |
+
+
+<a id="bbElecFunc"></a>
+## 9. bbElecFunc
+Ball-Berry: dependence of photosynthesis on PAR
```

Comment:
I think it would be helpful to spell out what `PAR` is.

Reply:
Agreed - but this is only meant as a skeleton so I had a place to document the `tmZoneInfo` option, which is new. We should expand all of these. I opened an issue for this #325